scripts/python/PyAutoscoper/connect.py

```diff
@@ -5,6 +5,25 @@
 EXPECTED_SERVER_VERSION = 1
 
 
+class CostFunction(Enum):
+    """Enum for the different cost functions available in PyAutoscoper."""
+
+    NCC = 0
+    NORMALIZED_CROSS_CORRELATION = 0
+    BONE_MODEL = 0
+    SAD = 1
+    SUM_OF_ABSOLUTE_DIFFERENCES = 1
+    IMPLANT_MODEL = 1
```

Comment:
Instead of having cost function labelled  `IMPLANT_MODEL` or `BONE_MODEL`, I suggest we remove it and instead have a function like `costFunctionForModel()` returning the model type.